From: Marc Kleine-Budde <mkl@pengutronix.de>
Date: Thu, 17 Mar 2011 13:34:53 +0100
Subject: [PATCH] teach ncurses-config about sysroot

This patch teaches ncurses-config about sysroot

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
[ported from ncurses 6.1 to 6.2]
Signed-off-by: Robert Schwebel <rsc@pengutronix.de>
[ported from ncurses 6.3 to 6.4]
Signed-off-by: Christian Melki <christian.melki@t2data.com>
---
 misc/ncurses-config.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/misc/ncurses-config.in b/misc/ncurses-config.in
index c9f1cf809843..af37d68c022c 100644
--- a/misc/ncurses-config.in
+++ b/misc/ncurses-config.in
@@ -188,10 +188,10 @@ while [ $# -gt 0 ]; do
 	--cflags)
 		INCS="@PKG_CFLAGS@"
 		if [ "x@WITH_OVERWRITE@" = xno ]; then
-			INCS="$INCS -I${includesubdir}"
+			INCS="$INCS -I${SYSROOT}${includesubdir}"
 		fi
 		if [ "${includedir}" != /usr/include ]; then
-			INCS="$INCS -I${includedir}"
+			INCS="$INCS -I${SYSROOT}${includedir}"
 		fi
 		sed -e 's,^[ ]*,,' -e 's, [ ]*, ,g' -e 's,[ ]*$,,' <<-ENDECHO
 			$INCS
@@ -274,7 +274,7 @@ ENDECHO
 		echo "$INCS"
 		;;
 	--libdir)
-		echo "${libdir}"
+		echo "${SYSROOT}${libdir}"
 		;;
 	--mandir)
 		echo "${mandir}"
